public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -723,7 +723,9 @@ export class TempoDatasource extends DataSourceWithBackend<TempoQuery, TempoJson
     if (query.queryType === 'nativeSearch') {
       let result = [];
       for (const key of ['serviceName', 'spanName', 'search', 'minDuration', 'maxDuration', 'limit']) {
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
         if (query.hasOwnProperty(key) && query[key as keyof TempoQuery]) {
+          // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
           result.push(`${startCase(key)}: ${query[key as keyof TempoQuery]}`);
```

Comment:
PR looks good, wondering why we added these though?

Reply:
They were initially added because I was doing a similar thing and @academo suggested to use `eslint-disable-next-line @typescript-eslint/consistent-type-assertions` to remove betterer error https://github.com/grafana/grafana/pull/78468#discussion_r1400632291. So I implemented it for my change and also for these 2 occurrences of `key as keyof`. I can revert if needed, just let me know!